sdk/internal/internal.go

```diff
@@ -14,16 +14,7 @@
 
 package internal // import "go.opentelemetry.io/otel/sdk/internal"
 
-import (
-	"fmt"
-	"time"
-
-	"go.opentelemetry.io/otel"
-)
-
-// UserAgent is the user agent to be added to the outgoing
-// requests from the exporters.
-var UserAgent = fmt.Sprintf("opentelemetry-go/%s", otel.Version())
```

Comment:
Looking at the git history this variable was [added in the initial SDK](0f32efcd), [updated when the `otel` package was shuffled](3d6e8b50), but never referenced.

```terminal
$ git log --oneline -G'UserAgent' ./sdk
3d6e8b50 Move Version function and code from global to toplevel (#1330)
0f32efcd Trace sdk (#65)
```

It should be safe to remove it as it looks stale.